src/controller/LottoController.js

```diff
@@ -0,0 +1,68 @@
+import LottoService from '../service/LottoService.js';
+import InputView from '../view/InputView.js';
+import OutputView from '../view/OutputView.js';
+
+export default class LottoController {
+  #lottoService;
+  constructor() {
```

Comment:
`constructor` 앞에는 개행을 하는 것이 좋아보입니다! 필드와 생성자를 구분하기 위해서요 😁 어떻게 생각하시나용?
```suggestion
  #lottoService;
  
  constructor() {
```

Reply:
개행을 의미있게 사용하고자 했는데 놓친 부분이 많네요 ㅜㅜ
개행을 통해 분리하는 한다는 지예님 의견에 동의합니다!!  꼼꼼한 리뷰 감사합니다!